### PR TITLE
test: Clean-up turning actual result to yaml

### PR DESCRIPTION
**Pull request by fviernau**
See individual commits.

### CODE REVIEW

File: scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt
Comment (sschuberth) [inline scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt:1]: Commit message: Typo in "scannerIntegrationFunTest" (should start capitalized).

File: scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt [hunk 1]
```diff
@@ -80,10 +80,9 @@ class ScannerIntegrationFunTest : StringSpec() {
                 )
             )
 
-            val ortResult = scanner.scan(analyzerResultFile.readValue(), skipExcluded = false, emptyMap())
-            val result = yamlMapper.writeValueAsString(ortResult)
+            val result = scanner.scan(analyzerResultFile.readValue(), skipExcluded = false, emptyMap())
 
-            patchActualResult(result, patchStartAndEndTime = true) shouldBe expectedResult
+            patchActualResult(result.toYaml(), patchStartAndEndTime = true) shouldBe expectedResult
```
Comment (sschuberth) [inline scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt:85]: Note that there's more test code that calls `yamlMapper.writeValueAsString(...)`, though on other objects than `OrtResult`. I propose to either make clear in the commit message that this targets only `OrtResult`, or replace the other calls with `toYaml()`, too.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt (hunk 1 above)
Comment (fviernau) [reply scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt:85]: I provided the class name in the commit title to make the scope (this class) of the commit clear. This doesn't help?

Comment (sschuberth) [reply scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt:85]: Yes, sorry, that's enough.

**Review by sschuberth — APPROVED** [review]

File: scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt (hunk 1 above)
Comment (fviernau) [reply scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt:85]: Got used to commits which fix things across multiple tests I guess....no wonder why

**Pull request merged**